src/components/alert/alert.scss

```diff
@@ -89,3 +89,24 @@
     transform: scale(scale.$effect-shrink);
   }
 }
+
+.c-alert__icon {
+  align-items: center;
+  background-color: lighten(color.$base-orange, 30%);
+  border-bottom-left-radius: size.$border-radius-medium;
+  border-top-left-radius: size.$border-radius-medium;
+  color: color.$base-orange;
+  display: flex;
+  grid-area: icon;
+  justify-content: center;
+  width: ms.step(5);
```

Comment:
We should add a comment explaining how we came to this size since it's sort of a magic number. For example, if this was chosen based on the usual single-line height of the pattern, that's a good thing to make note of.